walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/TimeRange.java

```diff
@@ -183,4 +184,37 @@ public static TimeRange fromStartEnd(int start, int end, boolean inclusive) {
   public static TimeRange fromStartDuration(int start, int duration) {
     return new TimeRange(start, duration);
   }
+
+  /**
+   * Uses a branchless binary search to find the TimeRange with the biggest starting point,
+   * that is <= to the starting point of target.
+   * It searches the index of the target bit by bit, starting with the MSB and going
```

Comment:
nit: Here I'd expand the MSB and LSB abbreviations into whatever they stand for. This helps the reader.